### PR TITLE
[BACKLOG-10556]: Fix/Add/Rework unit tests in big-data-plugin\kettle-…

### PR DESCRIPTION
**Pull request by TatsianaKasiankova**
…plugins after deletion hadoop-20 shim

Moved and refactored unit tests from org.pentaho.hadoop.mapreduce.test.PentahoMapReduceIT to pentaho-hadoop-shims\common
Because they test methods in GenericTransCombiner, GenericTransReduce and PentahoMapRunnable and should be placed in the appropriate project - pentaho-hadoop-shims\common

### CODE REVIEW

**Comment by TatsianaKasiankova** [comment]
@dkincade Hi, please review the PR.
Wingman rules exclude changes in pentaho-hadoop-shims\common to check.
I've verified the changes locally and onhttp://ci.pentaho.com/job/pentaho-hadoop-shims-stepanovdg-fork1/181/ - everything is Ok.

File: common/test-src/resources/bad-injector-fields.ktr [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    </clusterschemas>
+  <created_user>-</created_user>
+  <created_date>2010&#47;08&#47;13 13:40:30.869</created_date>
+  <modified_user>-</modified_user>
+  <modified_date>2010&#47;08&#47;13 13:40:30.869</modified_date>
+  </info>
+  <notepads>
+  </notepads>
+  <connection>
+    <name>Hypersonic</name>
```
Comment (dkincade) [inline common/test-src/resources/bad-injector-fields.ktr]: This connection is not used in the transformation. Please remove it.

File: common/test-src/resources/bad-injector-fields.ktr [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+      <attribute><code>FORCE_IDENTIFIERS_TO_UPPERCASE</code><attribute>N</attribute></attribute>
+      <attribute><code>IS_CLUSTERED</code><attribute>N</attribute></attribute>
+      <attribute><code>PORT_NUMBER</code><attribute>9002</attribute></attribute>
+      <attribute><code>QUOTE_ALL_FIELDS</code><attribute>N</attribute></attribute>
+      <attribute><code>SUPPORTS_BOOLEAN_DATA_TYPE</code><attribute>N</attribute></attribute>
+      <attribute><code>USE_POOLING</code><attribute>N</attribute></attribute>
+    </attributes>
+  </connection>
+  <connection>
+    <name>MySQL</name>
```
Comment (dkincade) [inline common/test-src/resources/bad-injector-fields.ktr]: This connection is not used in the transformation. Please remove it.

**Comment by dkincade** [comment]
Please remove the unneeded connections from the transformations provided. These can only cause issues in the future.

**Comment by TatsianaKasiankova** [comment]
Closed. Please see the final PR: https://github.com/pentaho/pentaho-hadoop-shims/pull/390

**Pull request closed without merging**